README.adoc

```diff
@@ -1,25 +1,74 @@
 == Getting Started with Spring Cloud Task
-This guide walks you through an overview of Spring Cloud Task and the process of creating and launching short-lived
-batch application.
+This guide walks you through an overview of Spring Cloud Task and the process of creating and launching a short-lived application.
+
+== Starting with Spring Initializr
+
+You can use this https://start.spring.io/#!type=maven-project&language=java&platformVersion=3.0.0-RC2&packaging=jar&jvmVersion=17&groupId=com.example&artifactId=demo&name=demo&description=Demo%20project%20for%20Spring%20Boot&packageName=com.example.demo&dependencies=jdbc,cloud-task,mariadb,h2[pre-initialized project] and click Generate to download a ZIP file. This project is configured to fit the examples in this tutorial.
+
+To manually initialize the project:
+
+. Navigate to https://start.spring.io.
+This service pulls in all the dependencies you need for an application and does most of the setup for you.
+. Choose either Gradle or Maven and the language you want to use. This guide assumes that you chose Java.
+. Click *Dependencies* and select *Task*, *H2 Database*, *JDBC API*, and *MariaDB Driver*.
```

Comment:
Change `Task`, `H2 Database`, `JDBC API`, and `MariaDB Drivers` to be italic. Actions that one can take get bold, but choices that one can make get italicized. The idea is to differentiate the two so that the user knows what kind of thing to find.